framework_tvb/tvb/tests/framework/core/services/flow_service_test.py

```diff
@@ -160,20 +161,19 @@ def test_prepare_adapter(self):
         stored_adapter = dao.get_algorithm_by_module(TEST_ADAPTER_VALID_MODULE, TEST_ADAPTER_VALID_CLASS)
         interface = self.flow_service.prepare_adapter(stored_adapter)
         assert isinstance(stored_adapter, model_operation.Algorithm), "Something went wrong with valid data!"
-        assert "name" in interface[0], "Bad interface created!"
-        assert interface[0]["name"] == "test", "Bad interface!"
-        assert "type" in interface[0], "Bad interface created!"
-        assert interface[0]["type"] == "int", "Bad interface!"
-        assert "default" in interface[0], "Bad interface created!"
-        assert interface[0]["default"] == "0", "Bad interface!"
+        assert "name" in interface.KEY_NAME, "Bad interface created!"
+        assert "filterable" in interface.KEY_FILTERABLE, "Bad interface created!"
+        assert "type" in interface.KEY_TYPE, "Bad interface created!"
+        assert "value" in interface.KEY_VALUE, "Bad interface created!"
+        assert "default" in interface.KEY_DEFAULT, "Bad interface created!"
```

Comment:
These asserts are not useful anymore, we should reimplement the test for this method.